render/helpers.go

```diff
@@ -95,3 +119,25 @@ func Markdown(md string) template.HTML {
 	s := blackfriday.Run([]byte(fmt.Sprintf("%s", modifiedMarkdown.String())))
 	return template.HTML(s)
 }
+
+// Localise localises text based on a key
+func Localise(key string, language string, plural int, templateArguments ...string) string {
+	if key == "" {
```

Comment:
Would it be useful to log "no key provided"?

Reply:
Yes, have now added this too in the format of `"key {key}  not found in locale file"` Where key is the identification key used.